### PR TITLE
Release 1.99.0

### PR DESCRIPTION
**Pull request by fluiddot**
Release for Gutenberg Mobile 1.99.0

## Related PRs

- Gutenberg: https://github.com/WordPress/gutenberg/pull/52368
- WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/18747
- WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/21018

## Extra PRs that Landed After the Release Was Cut

No extra PRs yet. 🎉

### Rename Reusable blocks to Patterns
- **PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5885

### Fix a crash related to Reanimated when closing the editor
- **PR:** https://github.com/wordpress-mobile/gutenberg-mobile/pull/5938
- **Issue:** https://github.com/wordpress-mobile/WordPress-iOS/issues/20499

## Test plan

Once the installable builds of the main apps are ready, perform a quick smoke test of the editor on both iOS and Android to verify it launches without crashing. We will perform additional testing after the main apps cut their releases.

## Release Submission Checklist

- [ ] Verify Items from test plan have been completed
- [x] Check if `RELEASE-NOTES.txt` is updated with all the changes that made it to the release. Replace `Unreleased` section with the release version and create a new `Unreleased` section.
- [x] Check if `gutenberg/packages/react-native-editor/CHANGELOG.md` is updated with all the changes that made it to the release. Replace `## Unreleased` with the release version and create a new `## Unreleased`.
- [x] Bundle package of the release is updated.

### CODE REVIEW

File: bundle/ios/GutenbergNativeTranslations.swift [hunk 1]
```diff
@@ -196,6 +195,7 @@ private func dummy() {
     _ = NSLocalizedString("Line Height", comment: "")
     _ = NSLocalizedString("Link inserted", comment: "")
     _ = NSLocalizedString("Link Rel", comment: "")
+    _ = NSLocalizedString("Link Settings", comment: "")
```
Comment (fluiddot) [inline bundle/ios/GutenbergNativeTranslations.swift:198]: This string is no longer used in the web version after [this change](https://github.com/WordPress/gutenberg/pull/51712/files#diff-4b02749cf17631881372540e1ff4bed69a8ce72fc214e7208660ae1c1abffab0L32).

File: bundle/ios/GutenbergNativeTranslations.swift [hunk 2]
```diff
@@ -10,8 +10,7 @@ private func dummy() {
     _ = NSLocalizedString("%s block, newly available", comment: "translators: Newly available block name. %s: The localized block name")
     _ = NSLocalizedString("%s block. Empty", comment: "translators: accessibility text for the media block empty state. %s: media type")
     _ = NSLocalizedString("%s block. This block has invalid content", comment: "translators: accessibility text for blocks with invalid content. %d: localized block title")
-    _ = NSLocalizedString("%s converted to regular block", comment: "translators: %s: name of the reusable block")
-    _ = NSLocalizedString("%s converted to regular blocks", comment: "translators: %s: name of the reusable block")
+    _ = NSLocalizedString("%s detached", comment: "translators: %s: name of the synced block")
```
Comment (fluiddot) [inline bundle/ios/GutenbergNativeTranslations.swift:13]: All strings related to reusable blocks have been updated due to [this expected change](https://github.com/WordPress/gutenberg/pull/51704).

**Review by geriux — APPROVED** [review]
LGTM! 🚀

**Comment by fluiddot** [comment]
## 🟢   Writing Flow (iOS)
**Build**: 22.8.0.0 (TestFlight)
**Device**: iPhone 11 (iOS 15.4.1)

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/multiline-components.md#tc001) - Multiline components
  - [x] Quote block
  - [x] Verse block
  - [x] Preformatted block
  - [ ] Code block (DEV only)
  - [x] Pullquote block
#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [ ] Paragraph
  - [ ] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [ ] Paragraph
  - [ ] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [ ] Paragraph
  - [ ] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [ ] Paragraph
  - [ ] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [ ] Paragraph
  - [ ] Heading
#### Undo / Redo - Test Cases
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc002) - Undo/redo text

**Comment by fluiddot** [comment]
## 🟢   Unsupported Block Tests (iOS)
**Build**: 22.8.0.0 (TestFlight)
**Device**: iPhone 11 (iOS 15.4.1)

- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

**Comment by fluiddot** [comment]
## 🟢  Release Changes (iOS)
**Build**: 22.8.0.0 (TestFlight)
**Device**: iPhone 11 (iOS 15.4.1)

- [x] [Rename Reusable blocks to Patterns](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5885)
- [x] [Fix a crash related to Reanimated when closing the editor](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5938)

**Comment by fluiddot** [comment]
## 🔴   Functionality Test Suite 5 (iOS)
**Build**: 22.8.0.0 (TestFlight)
**Device**: iPhone 11 (iOS 15.4.1)

- 🔴 I encountered [this crash](https://github.com/wordpress-mobile/gutenberg-mobile/issues/5925) when removing the File block. It's affecting a low volume of users (around 0.01%).

Video block - 1
- [x] Video block - Insert video from device (failing) - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc001)
- [x] Video block - Close/Re-open post with an ongoing video upload - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc004)

Video block - 2
- [x] Video block - Insert video from device (cancel) - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc002)
- [x] Video block - Close post with an ongoing video upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc005)
- [x] Video block - Add Caption - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/video.md#tc003)

File block - 1
- [x] File block - Upload completes successfully from the device library - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc001)
- [x] File block - Insert completes successfully from the WordPress Media library - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc002)
- [x] File block - Upload fails from the device library - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc003)
- [x] File block - Stop the file upload before it completes - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc004)

File block - 2
- [x] File block - Close the post with an ongoing file upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc005)
- [x] File block - Replace the selected files - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc006)
- [x] File block - Change the alignment in the File Block - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc007)
- [x] File block - The Link To Block Setting should open - [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc008)

File block - 3
- [x] File block - Open link in new tab - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc009)
- [x] File block - Show/hide download button (Block Settings only) - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc010)
- [x] File block - Copy file URL - [TC011](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc011)
- [x] File block - Close/Re-open post with an ongoing file upload - [TC012](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/file.md#tc012)

Audio block - 1
- [x] Audio block - Upload completes successfully from the device library - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc001)
- [x] Audio block - Insert completes successfully from the WordPress Media library - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc002)

Audio block - 2
- [x] Audio block - Upload fails from the device library - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc003)
- [x] Audio block - Stop the audio file upload before it completes - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc004)

Audio block - 3
- [x] Audio block - Close the post with an ongoing audio file upload - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc005)
- [x] Audio block - Replace the selected files - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc006)
- [x] Audio block - Change the alignment in the Audio Block - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc007)

Audio block - 4
- [x] Audio block - Autoplay setting - [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc008)
- [x] Audio block - Loop setting - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc009)
- [x] Audio block - Close/Re-open post with an ongoing audio file upload - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/audio.md#tc010)

Reusable block - 1
- [x] Reusable block - Add block - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/reusable.md#tc001)
- [x] Reusable block - Convert to regular blocks - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/reusable.md#tc002)
  - 🔴  I also encountered [this crash](https://github.com/wordpress-mobile/gutenberg-mobile/issues/5925) when detaching a pattern. It didn't happen consistently but I experienced it several times.

Reusable block - 2
- [x] Reusable block - Deleted reusable block - [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/reusable.md#tc003)
- [x] Reusable block - Recursive block rendering - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/reusable.md#tc004)

Embed block - 1
- [x] Embed block - Inline previews are rendered correctly for the top 5 most-used providers - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/embed.md#tc001)

Embed block - 2
- [x] Embed block - Inline previews are rendered accordingly for each alignment option - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/embed.md#tc002)

**Comment by dcalhoun** [comment]
## 🟢 Writing Flow (Android)
**Build**: 22.8-rc-1 (Google Play)
**Device**: Samsung Galaxy S20 5G FE (Android 13)

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/multiline-components.md#tc001) - Multiline components
  - [x] Quote block
  - [x] Verse block
  - [x] Preformatted block
  - [ ] Code block (DEV only)
  - [x] Pullquote block
#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading
#### Undo / Redo - Test Cases
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/undo-redo.md#tc002) - Undo/redo text

**Comment by dcalhoun** [comment]
## 🟢 Unsupported Block Tests (Android)
**Build**: 22.8-rc-1 (Google Play)
**Device**: Samsung Galaxy S20 5G FE (Android 13)

- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc001) - User can edit unsupported blocks on Simple WP.com sites
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc002) - User can discard edits to an unsupported block on Simple WP.com sites
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc003) - Editing unsupported blocks is allowed on Gutenberg-enabled Atomic sites
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc004) - Editing unsupported blocks is disallowed on Classic-enabled Atomic sites
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc005) - Editing unsupported blocks is enabled on self-hosted sites accessed via Jetpack
- [x] [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/unsupported-block-editing.md#tc006) - Editing unsupported blocks is disallowed on self-hosted sites access via their own username and password

**Comment by dcalhoun** [comment]
## 🟢 Release Changes (Android)
**Build**: 22.8-rc-1 (Google Play)
**Device**: Samsung Galaxy S20 5G FE (Android 13)

- [x] [Rename Reusable blocks to Patterns](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5885)
- [ ] ~[Fix a crash related to Reanimated when closing the editor](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5938)~ iOS only

**Comment by dcalhoun** [comment]
## 🟡 Functionality Test Suite 4 (Android)
**Build**: 22.8-rc-1 (Google Play)
**Device**: Samsung Galaxy S20 5G FE (Android 13)

### DarkMode - 2

- [x] Spacer block - Dark mode - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/darkmode.md#tc004)
- [x] Buttons block - Dark mode - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/darkmode.md#tc005)

### Group - 2

- [x] Group - Navigation up button works as expected - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc006)
- [x] Group - Navigation down works according to parent-first approach - [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc007)
- [x] Group - Cross navigation between blocks works as expected - [TC008](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc008)
- [x] Group - Check if in DarkMode all components gets proper colors - [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc009)
- [x] Group - Check if nested Placeholder block can be replaced - [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/group.md#tc010)

### Buttons - 4

- [x] Buttons block - The newly created Buttons has set background color to the theme-default color in preview - [TC017](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc017)

### Buttons - 5

- [x] Buttons block - Render theme colors - [TC015](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc015)
- [x] Buttons block - Settings: Change Button border radius - [TC011](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc011)

### Buttons - 7

- [x] Buttons block - Toolbar link button is active when Button has link - [TC018](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/buttons.md#tc018)

### Editor Theme - 2
- [x] Custom Gradients - Check that custom gradients load in the editor - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc004)
- [x] Offline Support - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/editor-theme.md#tc005)

### Color Settings - 2
- [x] Pressing color swatch - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc004)
- [x] Customize gradient color - [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc005)
    - ⚠️ Encountered preexisting issue: https://github.com/WordPress/gutenberg/issues/37199
- [x] Custom color picker - [TC006](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/color-settings.md#tc006)

### VideoPress - 1
- [x] VideoPress block - Upload a video - [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc001)
- [x] VideoPress block - Add Caption - [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc002)
- [x] VideoPress block - Close post with an ongoing video upload - [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/jetpack/videopress.md#tc004)

**Pull request merged**